lib/Conversion/TorchToArith/TorchToArith.cpp

```diff
@@ -374,6 +374,9 @@ class ConvertTorchToArith : public ConvertTorchToArithBase<ConvertTorchToArith>
     target.addIllegalOp<AtenDivFloatOp>();
     patterns.add<ConvertAtenBinaryOp<AtenDivFloatOp, arith::DivFOp>>(
         typeConverter, context);
+    target.addIllegalOp<AtenDivIntOp>();
+    patterns.add<ConvertAtenBinaryOp<AtenDivIntOp, arith::DivFOp>>(
```

Comment:
I don't think this is right. The `arith::DivFOp` takes floats as arguments, while the `AtenDivIntOp` takes ints as arguments. I would instead of doing this do a decomposition of `AtenDivIntOp` into `AtenDivFloatOp`, inserting casts to turn the inputs into floats.